omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx.conf.erb

```diff
@@ -73,6 +73,12 @@ http {
   }
   <%- end -%>
 
+  <% if node['private_chef']['data_collector']['root_url'] && node['private_chef']['data_collector']['token'] -%>
+  upstream data-collector {
+    server <%= URI.parse(node['private_chef']['data_collector']['root_url']).host %>:<%= URI.parse(node['private_chef']['data_collector']['root_url']).port %>;
```

Comment:
If `token` were set but no `root_url` were set, nginx woudl fail to start. Maybe modify the conditional above to check for both?